docs/src/pages/system/basics/ArrayApi.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+
+function ArrayApi() {
+  return <Box fontSize={[12, 18, 24, 36, 72]}>Array API</Box>;
```

Comment:
I think that 3 breakpoints are enough to make our point, same for the other demos:
```suggestion
  return <Box fontSize={[12, 18, 24]}>Array API</Box>;
```

If you want to test 5 breakpoints, a test case would be better than leaking it to the educational content that the demos.